Release v1.8.0 (#236)
airbrake/pybrake: changelog.md

## [Unreleased]

## [1.8.0] - 2022-09-13

### Added

> - Add support for Pycnic framework with sample example

Reviewer: ```suggestion
- Added support for the Pycnic framework with a sample example
```

> - Add support for Starlette framework with sample example

Reviewer: ```suggestion
- Added support for the Starlette framework with a sample example
```

### Changes

> - Receive notices and APM events and periodically sends them

Reviewer: ```suggestion
- Receives notices and APM events and periodically sends them
```

>   asynchronously in the background (1 minutes). Up to 100 backlog items for

Reviewer: ```suggestion
  asynchronously in the background (1 minute). Up to 100 backlog items for
```

>   notice and APM are Performance (Shared for Routes, Route breakdown, query,

Reviewer: ```suggestion
  notice and APM Performance (Shared for Routes, Route breakdown, query,
```

  queue stats). In the event that the backlog queue becomes full, the item
  will be replaced and an error will be recorded.

> - Code refactored of FastAPI middleware as it is based on starlette framework

Reviewer: We shouldn't mention refactorings in the changelog. Our users don't care.
